### PR TITLE
Notorious Monster: Zmey Gorynych

### PR DESCRIPTION
**Pull request by CatsEyeXI**
**_I affirm:_**
- [x] I understand that if I do not agree to the following points by completing the checkboxes my PR will be ignored.
- [x] I understand I should leave resolving conversations to the LandSandBoat team so that reviewers won't miss what was said.
- [x] I have read and understood the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/LandSandBoat/server/blob/base/CODE_OF_CONDUCT.md).
- [x] I have _**tested my code and the things my code has changed**_ since the last commit in the PR and will test after any later commits.

## What does this pull request do?
Implements missing Notorious Monster: Zmey Gorynych.

Sources:
https://www.youtube.com/watch?v=mcbl4JS7bEg&pp=ygUSem1leSBnb3J5bnljaCBmZnhp
https://ffxiclopedia.fandom.com/wiki/Zmey_Gorynych
https://wiki.ffo.jp/html/18594.html

## Steps to test these changes
Fight the NM, notice it hits hard, in line with video, uses triple attach often, and scythe tail exclusively.

### CODE REVIEW

**Review by Xaver-DaRed — APPROVED** [review]

File: scripts/zones/Cape_Teriggan/mobs/Zmey_Gorynych.lua [hunk 1]
```diff
@@ -4,8 +4,21 @@
 -----------------------------------
 local entity = {}
 
+entity.onMobInitialize = function(mob)
+    -- This is a WAR mob, but does not use any double attack.
```
Comment (TeoTwawki) [inline scripts/zones/Cape_Teriggan/mobs/Zmey_Gorynych.lua]: All 3 of jp wiki bg and ffxiclopedia say its thf, jp wiki even claims thf+war with _both_ double and triple attack

> [攻撃力](https://wiki.ffo.jp/html/1766.html)が元々高い上に、運が悪いと[DA](https://wiki.ffo.jp/html/3922.html)＆[TA](https://wiki.ffo.jp/html/7800.html)による連続[攻撃](https://wiki.ffo.jp/html/11495.html)によって一瞬で[HP](https://wiki.ffo.jp/html/324.html)を削られることがある。[回避盾](https://wiki.ffo.jp/html/10075.html)が有効ではあるが、[攻撃](https://wiki.ffo.jp/html/11495.html)の[命中率](https://wiki.ffo.jp/html/226.html)も高く蝉は剥げやすい。[特殊技](https://wiki.ffo.jp/html/12837.html)は[サイズテール](https://wiki.ffo.jp/html/23463.html)しか使わず、終盤では怒涛のように使ってくる。[サイズテール](https://wiki.ffo.jp/html/23463.html)の直撃を受けると、[即死](https://wiki.ffo.jp/html/10818.html)の可能性が高く注意が必要。

which says 

> Not only does it have high attack power, but if you're unlucky, continuous attacks from DA & TA can instantly reduce its HP. Although the evasion shield is effective, the hit rate of attacks is high and the cicadas are easy to peel off. The only special technique he uses is Size Tail, and he uses it like a wave in the final stages. If you receive a direct hit from a size tail, there is a high possibility of instant death, so be careful.

Comment (TeoTwawki) [reply scripts/zones/Cape_Teriggan/mobs/Zmey_Gorynych.lua]: my point isn't that it has double (wiki's can be wrong!), its that its likely NOT a war

Comment (TeoTwawki) [reply scripts/zones/Cape_Teriggan/mobs/Zmey_Gorynych.lua]: so. I only just now noticed its already set thf/thf in pools and feel silly for not just telling you it doesn't have any DA to have to zero.

**Review by TeoTwawki — APPROVED** [review]
@Xaver-DaRed says he'll follow up on the DA

**Review by TeoTwawki — APPROVED** [review]
reeeee-re-approved

**Comment by TeoTwawki** [comment]
:x

I causeded da merge conflicts.

**Comment by CatsEyeXI** [comment]
> :x
> 
> I causeded da merge conflicts.

fixed... i think

**Pull request merged**